### PR TITLE
x86: don't build ext4 images

### PR DESCRIPTION
**Pull request by lemoer**
Since we're discarding the ext4 images anyways, we now stop building 
them as well and save a few seconds of build time.

### CODE REVIEW

**Comment by NeoRaider** [comment]
`filters.yml` is generated by `scripts/update-ci.sh`, so your changes will get lost with the next update...

**Comment by lemoer** [comment]
@NeoRaider Thanks. Updated.

File: contrib/actions/generate-target-filters.py [hunk 1]
```diff
@@ -31,6 +31,12 @@
         f"targets/{target}"
     ] + common
 
+    if target.startswith('x86-'):
+        _filter[target].append('targets/x86.inc')
+
+    if target.startswith('bcm27xx-'):
+        _filter[target].append('targets/bcm27xx.inc')
```
Comment (NeoRaider) [inline contrib/actions/generate-target-filters.py:38]: Hmm... on the one hand I'd like to have a more generic solution like the following (untested):
```py
if '-' in target:
    _filter[target].append('targets/{}.inc'.format(target.split('-')[0]))
```

On the other hand, we are building more targets than necessary now: `x86-geode` does not include `x86.inc`.

**Comment by lemoer** [comment]
@NeoRaider Ok. Updated.

File: contrib/actions/generate-target-filters.py [hunk 2]
```diff
@@ -23,6 +25,13 @@
 
 _filter = dict()
 
+# INCLUDE_PATTERN matches:
+# include '...'
+# include "..."
+# include("...")
+# include('...')
+INCLUDE_PATTERN = "^ *include[\(\ ][\"'](.*)[\"']\)? *$"
```
Comment (NeoRaider) [inline contrib/actions/generate-target-filters.py:33]: I propose the following pattern:
```suggestion
INCLUDE_PATTERN = "^\\s*include *\\(? *[\"']([^\"']+)[\"']"
```
- Allow any leading whitespace (in Gluon we usually use tabs)
- Allow any trailing string after the closing quote (the code might have a comment)
- Make whitespace matching between `include` and the opening quote more flexible
- Fix number of backslashes in a few places
- Only allow non-quote characters between the quotes, require at least one character
I have only tested this on a Python shell and not with the Gluon code, but it seems to work as expected.

**Comment by lemoer** [comment]
@NeoRaider Ok. Updated again.

**Pull request merged**